packages/notifi-react-card/lib/hooks/useUnreadState.ts

```diff
@@ -22,6 +22,10 @@ export const useUnreadState = () => {
     () => (unreadNotificationCount > 0 ? true : false),
     [unreadNotificationCount],
   );
+
+  const updateNotificationCount = () => {
```

Comment:
considering this is setting to 0, I think it should be called clearNotificationCount